#!/usr/bin/env python
PACKAGE = "rm_gimbal_controllers"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("q_x_pos", double_t, 0, "", 100, 0, 1000)
gen.add("q_y_pos", double_t, 0, "", 100, 0, 1000)
gen.add("q_z_pos", double_t, 0, "", 100, 0, 1000)
gen.add("q_yaw_pos", double_t, 0, "", 100, 0, 1000)
gen.add("q_x_vel", double_t, 0, "", 100, 0, 1000)
gen.add("q_y_vel", double_t, 0, "", 100, 0, 1000)
gen.add("q_z_vel", double_t, 0, "", 100, 0, 1000)
gen.add("q_yaw_vel", double_t, 0, "", 100, 0, 1000)

gen.add("r_x_pos", double_t, 0, "", 100, 0, 1000)
gen.add("r_y_pos", double_t, 0, "", 100, 0, 1000)
gen.add("r_z_pos", double_t, 0, "", 100, 0, 1000)
gen.add("r_yaw_pos", double_t, 0, "", 100, 0, 1000)
gen.add("r_x_vel", double_t, 0, "", 100, 0, 1000)
gen.add("r_y_vel", double_t, 0, "", 100, 0, 1000)
gen.add("r_z_vel", double_t, 0, "", 100, 0, 1000)
gen.add("r_yaw_vel", double_t, 0, "", 100, 0, 1000)

exit(gen.generate(PACKAGE, "kalman", "Kalman"))
